dataset/src/test/scala/frameless/forward/FilterTests.scala

```diff
@@ -0,0 +1,15 @@
+package frameless
+package forward
+
+import org.scalacheck.Prop
+import org.scalacheck.Prop._
+
+class FilterTests extends TypedDatasetSuite {
```

Comment:
For me, it would be ok to keep these tests among tests in `frameless` package. There is already `FilterTests` that does very similar things